{"sections":[],"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"text":"Inherited from ","type":"text"},{"code":"LocalizedError.errorDescription","type":"codeVoice"},{"text":".","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openai\/apierror\/errordescription"]}],"metadata":{"role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"externalID":"s:6OpenAI8APIErrorV16errorDescriptionSSSgvp","roleHeading":"Instance Property","extendedModule":"OpenAI","title":"errorDescription","modules":[{"name":"OpenAI"}],"symbolKind":"property"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations"]]},"identifier":{"url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription","interfaceLanguage":"swift"},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"? { ","kind":"text"},{"kind":"keyword","text":"get"},{"text":" }","kind":"text"}],"languages":["swift"]}]}],"references":{"doc://OpenAI/documentation/OpenAI/APIError/errorDescription":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror\/errordescription","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription"},"doc://OpenAI/documentation/OpenAI/APIError/LocalizedError-Implementations":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror\/localizederror-implementations","title":"LocalizedError Implementations","role":"collectionGroup","kind":"article","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations"},"doc://OpenAI/documentation/OpenAI":{"type":"topic","abstract":[],"role":"collection","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","title":"OpenAI","url":"\/documentation\/openai","kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror","title":"APIError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"APIError","kind":"identifier"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","navigatorTitle":[{"kind":"identifier","text":"APIError"}]}}}